src/linker/Linker/MessageCode.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Mono.Linker {
+	public enum MessageCode {
+		L100, // Error
+		L400, // Warning
+		L900, // Info.
```

Comment:
Did you talk to the Roslyn about the diagnostic IDs? We need to make sure that the codes we're about to add are unique and ideally that they follow some kind of pattern.

Reply:
They are still debating which way to go on this. For now, I simply used `IL` as a prefix.